### DISCUSSION TITLE
Simplisafe resolution Hombridge

### DISCUSSION THREAD

**Discussion by chiachs** [homebridge-simplisafe3/homebridge-simplisafe3#380] [Q&A]
@shamoon  Hi. I just installed the plugin and got the video stream to work using a solution you gave in another thread of adding -vf scale=720:480. is there anyway to get full resolution and not 480?

Thanks

**Comment by shamoon** [comment]
HomeKit is "picky" about the quality of incoming streams e.g. bitrate and presumably it rejects the native resolution (which is being transcoded still) at the native resolution. It sorta depends on your setup e.g. what device is doing transcoding (hardware acceleration etc) but essentially no, if its not working at the native resolution then scaling it down is the solution.

**Comment by chiachs** [comment]
Thanks

Alek B.
________________________________

**Reply by chiachs** [reply]
one more question if i may. does the plugin allow for talking back through the camera or doorbell?

**Reply by shamoon** [reply]
No, it does not https://github.com/homebridge-simplisafe3/homebridge-simplisafe3#supported-devices
